usr/src/lib/libsmbfs/smb/lgrep.awk

```diff
@@ -24,6 +24,7 @@
 # Use is subject to license terms.
 #
 # Copyright 2012 Milan Jurik. All rights reserved.
+# Copyright 2017 Nexenta Systems, Inc.  All rights reserved.
```

Comment:
This file makes me sad, as does your adding to it.

Reply:
This file was originally about this:
 # The mb_put/md_get functions are intentionally used both
 # with and without return value checks.  Not a concern.

I'm not aware of another way to tell the compiler that for some subset of functions,
I'd rather not hear about "return value is sometimes ignored".